src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -608,7 +613,8 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
         }
 
         internal static async Task InvokeAsync(IFunctionInvoker invoker, object[] invokeParameters, CancellationTokenSource timeoutTokenSource,
-            CancellationTokenSource functionCancellationTokenSource, bool throwOnTimeout, TimeSpan timerInterval, IFunctionInstance instance)
+            CancellationTokenSource functionCancellationTokenSource, bool throwOnTimeout, TimeSpan timerInterval, IFunctionInstance instance, ILogger logger = null,
+            JobHost jobHost = null, JobHostConfiguration config = null, IReadOnlyList<string> parameterNames = null)
```

Comment:
Yes, all those other locations need to be fixed up :) These are required parameters now. I see that all those places are in the tests - they just need to be updated.